cluster_tools/downscaling/downscaling_workflow.py

```diff
@@ -44,6 +44,7 @@ class DownscalingWorkflow(WorkflowBase):
     input_key = luigi.Parameter()
     scale_factors = luigi.ListParameter()
     halos = luigi.ListParameter()
+    int_to_uint = luigi.BoolParameter(default=False)
```

Comment:
Please also add the `dtype` arugment here to keep the signature fully compatible with `CopyVolume` (and pass it on to the appropriate functions).